### ISSUE TITLE
Pointed scans

### ISSUE THREAD

**Issue by mlincett** [icecube/skymap_scanner#237]
I have been chatting with @G-Sommani about how to implement "pointed" scans. This would allow to investigate, for example, different sky regions compared to the one to which the scan iterations converge.

The relevant logic is in [start_scan.py](https://github.com/icecube/skymap_scanner/blob/main/skymap_scanner/server/start_scan.py), specifically the `PixelsToReco` class.

The simplest way of doing this is to pass a pair of coordinates to `start_scan` and select only the first-NSIDE pixel pointed by those coordinates (instead of scanning/refining all the pixels). So the set of changes could be fairly minimal.

SkyDriver will have to support the additional input.

A logic of the type "scan a given radius around given coordinates" to determine the corresponding appropriate NSIDE sequence can be implemented in SkyMist.

**Comment by mlincett** [comment]
The logic is actually already implemented here:
https://github.com/icecube/skymap_scanner/blob/11081a4dfc644485ceff6e83c72ee83e7333dd82/skymap_scanner/server/pixels.py#L168-L174

It scans at max NSIDE around given coordinates and with a given radius. It is a bit more rigid than what I had in mind but also does not require any additional logic.

All there is to do is back-propagate the arguments up to `start_scan.py`.

**Comment by ric-evans** [comment]
> All there is to do is back-propagate the arguments up to `start_scan.py`.

Yup, I left this in. If you add the needed CI arguments, I can hook this up the Skydriver.
